Competition2017/src/competition/subsystems/hybrid/commands/BaseShooterBeltAndWheelCommand.java

```diff
@@ -0,0 +1,18 @@
+package competition.subsystems.hybrid.commands;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+import xbot.common.command.BaseCommand;
+
+public abstract class BaseShooterBeltAndWheelCommand extends BaseCommand {
+    
+    protected final ShooterBeltSubsystem shooterBeltSubsystem;
+    protected final ShooterWheelSubsystem shooterWheelSubsystem;
+    
+    public BaseShooterBeltAndWheelCommand(ShooterBeltSubsystem shooterBeltSubsystem, ShooterWheelSubsystem shooterWheelSubsystem) {
+        this.shooterBeltSubsystem = shooterBeltSubsystem;
+        this.shooterWheelSubsystem = shooterWheelSubsystem;
+        this.requires(shooterBeltSubsystem);
+        this.requires(shooterWheelSubsystem);
```

Comment:
Oh, interesting issue... I think we should just remove the new base class and put the `shooterWheelSubsystem` field directly into the command.